aibtc-v1/chat.py

```diff
@@ -0,0 +1,171 @@
+import streamlit as st
+import logging
+from typing import Dict, Any
+from utils.session import init_session_state
+
+# Initialize session state
+init_session_state()
+
+# Set up Streamlit page
+st.set_page_config(page_title="CrewAI Chatbot", layout="centered")
+
+# Constants
+AVAILABLE_CREWS: Dict[str, Dict[str, Any]] = st.session_state.crew_mapping
+
+
+def load_custom_styles():
```

Comment:
You can grab the code from the main app for the custom styles that were set, if needed.